<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="0" hostname="IXDM7J5B3" name="cucumberoptions.TestRunner" tests="9" failures="1" timestamp="2022-05-27T17:15:59 IST" time="134.928" errors="0">
  <testcase name="runScenario" time="12.908" classname="cucumberoptions.TestRunner"/>
  <system-out/>
  <testcase name="runScenario" time="11.232" classname="cucumberoptions.TestRunner"/>
  <system-out/>
  <testcase name="runScenario" time="11.885" classname="cucumberoptions.TestRunner"/>
  <system-out/>
  <testcase name="runScenario" time="11.328" classname="cucumberoptions.TestRunner">
    <failure type="java.lang.AssertionError" message="expected [Loan Request Processed] but found [Apply for a Loan]">
      <![CDATA[java.lang.AssertionError: expected [Loan Request Processed] but found [Apply for a Loan]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertEqualsImpl(Assert.java:140)
at org.testng.Assert.assertEquals(Assert.java:122)
at org.testng.Assert.assertEquals(Assert.java:629)
at org.testng.Assert.assertEquals(Assert.java:639)
at stepdefinition.RequestLoan.verify_that_user_is_able_to_request_loan(RequestLoan.java:35)
at âœ½.Verify that user is able to request loan(file:///C:/Users/703316032/Eclipse/CucumberProject/resources/features/RequestLoan.feature:6)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <system-out/>
  <testcase name="runScenario" time="22.541" classname="cucumberoptions.TestRunner"/>
  <system-out/>
  <testcase name="runScenario" time="16.040" classname="cucumberoptions.TestRunner"/>
  <system-out/>
  <testcase name="runScenario" time="18.052" classname="cucumberoptions.TestRunner"/>
  <system-out/>
  <testcase name="runScenario" time="24.569" classname="cucumberoptions.TestRunner"/>
  <system-out/>
  <testcase name="runScenario" time="6.373" classname="cucumberoptions.TestRunner"/>
  <system-out/>
</testsuite> <!-- cucumberoptions.TestRunner -->
